perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -313,7 +269,11 @@ def _GetOptionDecoderConstructions(cls):
         'host_type': (option_decoders.StringDecoder,
                       {'default': 'n1-host-64-416'}),
         'num_vms_per_host': (option_decoders.IntDecoder, {'default': None}),
-        'min_cpu_platform': (option_decoders.StringDecoder, {'default': None})
+        'min_cpu_platform': (option_decoders.StringDecoder, {'default': None}),
+        'gpu_type': (option_decoders.EnumDecoder, {
+            'valid_values': GPU_TYPE_TO_INTERAL_NAME_MAP.keys(),
```

Comment:
I may be missing this, but is the superset (GPU_TYPE_TO_INTERAL_NAME_MAP.keys()) defined somewhere ?